flask_appbuilder/security/sqla/manager.py

```diff
@@ -232,6 +247,129 @@ def update_user(self, user):
     def get_user_by_id(self, pk):
         return self.get_session.query(self.user_model).get(pk)
 
+    def get_reset_password_hash(self, user_id):
+        return (
+            self.get_session.query(UserResetPassword)
+            .filter(UserResetPassword.id == user_id)
+            .scalar()
+        )
+
+    def check_expire_reset_password_hash(self, resetpw):
+        """
+            Returns True if the reset password hash has not been expired.
```

Comment:
nit: comment is too indentation (I know I was doing it like this, but it is wrong) :)

And btw thanks for fixing that!